### DISCUSSION TITLE
Deebot 710/700

### DISCUSSION THREAD

**Discussion by fbonhoff** [mrbungle64/ecovacs-deebot.js#111] [Q&A] [answered]
Has anyone got one of those running? I can get the status information but can't seem to get the robot to start cleaning...

I simply used the code provided in the usage section of the git and added 'vacbot.clean()' after a connection to the ecovacs server has been established.

I also tried vacbot.run('Clean'), but neither did anything.

I hope I am just stupid in anyway and i am misusing the code

**Comment by mrbungle64** [comment, marked as answer]
@fbonhoff 

Are you using the latest version?
I just fixed that issue for the Deebot 710 about 1 week ago

**Reply by fbonhoff** [reply]
I should be since i just installed it today, on npm -v ecovacs-deebot.js it says version 8.3.0.

**Reply by mrbungle64** [reply]
@fbonhoff 
You have to use the latest version from GitHub or 0.7.0-beta.3 from npm

```
npm i ecovacs-deebot@0.7.0-beta.3
```

**Reply by fbonhoff** [reply]
its working, thanks for the quick response and amazong repo! also sorry for being unexperienced and stupid...
